specification/communication/data-plane/Email/preview/2023-15-01-preview/CommunicationServicesEmail.json

```diff
@@ -0,0 +1,413 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationEmailClient",
+    "description": "Azure Communication Email Service",
+    "version": "2023-15-01-preview"
+  },
+  "paths": {
+    "/emails/{messageId}/status": {
```

Comment:
We have one subpath of `status`, does that mean we expect to ship `/emails/{messageId}` in the future which will hold the sent email message including content?
Will this service ever be able to receive email messages? Then we need an API design that distinguished inbox from outbox.

Reply:
No change here.